src/hotspot/share/jeandle/jeandleAbstractInterpreter.hpp

```diff
@@ -38,6 +38,7 @@
 #include "memory/allocation.hpp"
 #include "memory/universe.hpp"
 #include "utilities/bitMap.inline.hpp"
+#include "jeandle/jeandleType.hpp"
```

Comment:
Jeandle's header file should added before hotspot's headers.
Refer to https://github.com/jeandle/jeandle-jdk/blob/main/jeandle-docs/development-guide.md#header-files-in-hotspot-code

Reply:
fixed